### PR TITLE
feat: replat 6371 tilestar new design

### PR DESCRIPTION
**Pull request by parkjh**
Screenshot 2019-05-23 at 16 06 52
Screenshot 2019-05-23 at 16 07 07

### CODE REVIEW

File: packages/edition-slices/src/tiles/shared/tile-link.js [hunk 1]
```diff
@@ -1,28 +1,48 @@
...
+  withStar,
+  starStyle,
   tile: {
     article: { id, url }
   }
 }) => (
   <Link linkStyle={style} onPress={() => onPress({ id, url })} url={url}>
     {children}
+    {withStar &&
+      Platform.OS !== "ios" && (
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/tiles/shared/tile-link.js]: We don't need a platform check, we already have a feature availability check (checks if NativeModules.SectionEvents.getSavedArticles method is present)

Comment (tuncaulubilge) [reply packages/edition-slices/src/tiles/shared/tile-link.js]: You can create a shared util function like `isSaveSupported` and do the check there

File: packages/edition-slices/src/tiles/shared/styles/index.js [hunk 2]
```diff
@@ -8,23 +6,26 @@ const horizontalStyles = {
...
 };
 
+const tileStar = {
+  position: "absolute",
+  right: 10,
+  bottom: 5
+};
+
+const starPadding = {
+  paddingBottom: 25
```
Comment (aronshamash) [inline packages/edition-slices/src/tiles/shared/styles/index.js]: should use `spacing`

File: packages/edition-slices/src/tiles/shared/styles/index.js [hunk 3]
```diff
@@ -8,23 +6,26 @@ const horizontalStyles = {
...
-    alignItems: "flex-end",
-    flex: 1,
-    marginTop: -5,
-    width: "25%"
   }
 };
 
+const tileStar = {
+  position: "absolute",
+  right: 10,
```
Comment (aronshamash) [inline packages/edition-slices/src/tiles/shared/styles/index.js]: even though not margin or padding `spacing` may be appropriate ?

Comment (parkjh) [reply packages/edition-slices/src/tiles/shared/styles/index.js]: @aronshamash nope this is reliant on the padding of the parent tile-link, in order to make the absolute position more consistent...

**Comment by parkjh** [comment]
Screenshot 2019-05-24 at 13 47 44

**Comment by parkjh** [comment]
Screenshot 2019-05-24 at 13 48 28

File: packages/edition-slices/src/tiles/shared/styles/index.js (hunk 3 above)
Comment (tuncaulubilge) [reply packages/edition-slices/src/tiles/shared/styles/index.js]: Anything spacing related should use spacing, including this

File: packages/edition-slices/src/tiles/shared/utils.js [hunk 4]
```diff
@@ -1,3 +1,7 @@
+import { NativeModules } from "react-native";
+
+const { getSavedArticles } = NativeModules.SectionEvents;
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/tiles/shared/utils.js]: Can you test this on web? You might need a null check since NativeModules might be null

**Comment by parkjh** [comment]
https://media.giphy.com/media/JJhiRdcYfcokU/giphy.gif

**Comment by parkjh** [comment]
ezgif-4-1aa9cb7ba13b

**Pull request merged**